dnf/cli/commands/alias.py

```diff
@@ -157,6 +158,8 @@ def list_alias(self, cmd):
             print(_("Alias %s='%s'") % (cmd, " ".join(args)))
 
     def run(self):
+        dnf.util.is_container()
```

Comment:
Do we want to spread the message to all commands including commands from plugins (dnf-plugins-core, extras, third party), or do we want to be selective?

Reply:
I have gone back and forth on this and I think it's mostly wasted effort to be selective right.  I think it's a better idea to do a blanket warning and then shift the focus to incorporating rpm-ostree functionality directly in to native dnf (or a dnf core plugin) where appropriate and then eventually we can just drop this change.  I view this change as a temporary thing just to make it nicer for users.